app.libretile.Libretile.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: app.libretile.Libretile
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: libretile.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+build-options:
+    build-args:
+      -    --share=network
+modules:
+  - name: libresprite
+    buildsystem: simple
+    
+    build-commands:
+       - pip3 install --prefix=/app pillow
+       - git clone https://gitlab.com/celleron56/wavefunctioncollabseterrain 
```

Comment:
Is this really necessary considering that this is part of the sources too? Also, I'm not sure it will work on the builders.

Reply:
it will no.t There is no network access.

Reply:
`build-options:
    build-args:
      -    --share=network`
this code should deal with that     also i am not sure  where the sources get downloaded

Reply:
> `build-options: build-args: - --share=network`
> this code should deal with that also i am not sure where the sources get downloaded

This option is not respected on Flathub's builders.

Reply:
It's actually not allowed. The build will fail if it encounter it.

Reply:
ok  i removed it